features/step_definitions/team_steps.rb

```diff
@@ -32,3 +34,21 @@
 Then(/^I click on team title$/) do
   find('#team-name').click
 end
+
+Given('the following users are registered with teams:') do |table|
+  table.hashes.each do |row|
+    team = Team.find_by(name: row[:team])
+    team ||= FactoryBot.create(:team, name: row[:team])
+    user = FactoryBot.create(:user, row.slice('name', 'email'))
+    FactoryBot.create(:user_team, user: user, team: team, role: row[:role])
+  end
+end
+
+Given('team settings page') do
+  visit teams_path
+  page.save_screenshot(full: true)
```

Comment:
remove it.
Also we can add support step:
```
Then('I make screenshot') do
  page.save_screenshot(full: true)
end
```
And you can easy add it between steps.

Reply:
Also we can also add gem to automatically open this screenshot.
`page.save_and_open_screenshot(full: true)`